ods_ci/tests/Tests/0100__platform/0101__deploy/0104__operators/0104__rhods_operator/0108__rhods_operator_logs_verification.robot

```diff
@@ -18,19 +18,19 @@ Suite Teardown    RHOSi Teardown
 
 
 *** Variables ***
-${namespace}           ${OPERATOR_NAMESPACE}
 ${regex_pattern}       level=([Ee]rror).*|([Ff]ailed) to list .*
 
 
 *** Test Cases ***
 Verify RHODS Operator log
-   [Tags]  Sanity    Tier1
```

Comment:
As far as I understand, this checks whether the operator pod starts without any errors in the log, right? Then I would make this a part of Sanity tests.

Reply:
I agree with you, let me change it